src/options.js

```diff
@@ -43,6 +44,7 @@ options.Options = {
  */
 options.DEFAULTS_ = {};
 options.DEFAULTS_[options.Options.BADGE_TEXT_SHOWN] = true;
+options.DEFAULTS_[options.Options.NOTIFICATION_SHOW] = false;
```

Comment:
Seems fine to me to enable notifications by default; what do you think?

Reply:
Yeah, I think we can do that.